ui/src/plugins/logger.js

```diff
@@ -0,0 +1,18 @@
+export default {
+  install(Vue) {
+    function log(type, message, ...args) {
+      if (process.env.NODE_ENV === "production") {
+        return;
```

Comment:
I think for production we might want all messages but `debug` ones. If the app is reporting an error, users can give us useful information.